linetools/spectra/utils.py

```diff
@@ -35,25 +35,18 @@ def from_spec1d(cls, spec1d):
 
     #### ###############################
     #  Normalize
-    def normalize(self, iconti, verbose=False, no_check=False):
+    def normalize(self, conti, verbose=False, no_check=False):
```

Comment:
That is right.  I fear those carried over
from something else..

X

On Jun 21, 2015, at 1:17 PM, Neil Crighton <notifications@github.com> wrote:

> In linetools/spectra/utils.py:
> 
> > @@ -35,25 +35,18 @@ def from_spec1d(cls, spec1d):
> >  
> >      #### ###############################
> >      #  Normalize
> > -    def normalize(self, iconti, verbose=False, no_check=False):
> > +    def normalize(self, conti, verbose=False, no_check=False):
> The changes in this file have nothing to do with the LineList class, right? (just checking).
> 
> —
> Reply to this email directly or view it on GitHub.
> 
> !DSPAM:826,55871bd91162707519558!

----------------------------------------------
J. Xavier Prochaska
UCO/Lick Observatory
1156 High St.
UC Santa Cruz
Santa Cruz, CA 95064

http://www.ucolick.org/~xavier/
http://imps.ucolick.org/
http://www.qpqsurvey.org/
http://k1dm3.ucolick.org/
http://www.ucolick.org/~xavier/IDL/index.html
https://github.com/profxj/xastropy

831-459-2135 (Direct)
831-459-2991 (UCO/Lick Main)
831-459-5244 (Fax)